packages/node_modules/@ciscospark/react-redux-spark/src/component.test.js

```diff
@@ -7,14 +7,44 @@ import {createMockStore} from '@ciscospark/react-redux-spark-fixtures';
 import SparkComponent from './component';
 
 describe('spark component', () => {
-  it('renders correctly', () => {
-    const store = createMockStore();
-    const component = renderer.create(
+  let sparkInstance, store, component;
+
+  beforeEach(() => {
```

Comment:
In order for me to test the components methods, I need to use the `enzyme` library. where its not used in this project. unless you guys know other ways to test the methods without enzyme.

Reply:
I think if you export the unconnected class you should be able to test a `new` instance of it without using enzyme. I'm not opposed to adding enzyme tests though.